Assets/HoloToolkit/Utilities/Plugins/README.md

```diff
@@ -0,0 +1 @@
+This folder is for the Externals.zip that should alway be up to date.
```

Comment:
Why are we doing this? Just because we are unable to add the externals folder into the asset package?
One of the main reasons for making it external was they are binary files that don't compile under Assets.
Adding a zip file with the same stuff does not help that cause.

I think we can come up with a better solution than a zip file under Assets. 
Just for solving the problem of mismatching SharingServer.exe binaries problem, we could always add a check in code in the Launch Sharing Service option to check version numbers. I think that would better than the zip.

What do you think?